challenge-030/izifresh/Perl 5/christmas

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env perl
```

Comment:
1. Script must show _only_ Christmas that will be at sunday. Not all future Christmases
2. Try to wrap "magic numbers" into constants (`perldoc constant`), and your code will become more readable.
Or you can try to use hash as "day number" <-> "day name" mapping
3. change dir name and add extensions:
challenge-030/izifresh/Perl 5/christmas 
challenge-030/izifresh/perl5/christmas.pl
4. Stick with single indenting style.

Reply:
My bad @izifresh. Solutions must be named ch-1.pl and ch-2.pl